pep001-zenhub.md

```diff
@@ -0,0 +1,77 @@
+# PEP001 - ZenHub for TGDP Product Management
+
+## Proposed By
+
+Aaron Peters
+
+
+## State
+
+- [x] Draft
+- [ ] Under discussion
+- [ ] Final comment and voting
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+
+## Proposal Overview
+
+As The Good Docs project matures, we should strive to make regular releases to the community. Implementing a product management solution will provide transparency, and allow us to plan releases that regularly provide value while balancing the available bandwidth of project volunteers.
+
+
+## Motivation
+
+As The Good Docs Project grows, both in scope and in membership, a more formal “source-of-truth” for the org’s ongoing development will help ensure continued progress. This is particularly true given recent discussions of conducting meetings for individual Working Groups, during which decisions will be made and tasks taken up that would otherwise be difficult to coordinate using the default GitHub mechanisms.
+
+To this end, this proposal includes the supplement of the current work management system (GitHub) with enhanced project/product management capabilities.
+
+
+## Proposal
+
+In an effort to coordinate work across multiple working groups, this proposes the addition of [https://www.zenhub.com/](ZenHub) to the existing GitHub account for TGDP. This will bring multiple benefits including:
+
+* The ability to add multiple repositories to the Workspace.
+* The ability to define multiple projects for the Workspace.
+* The ability to group related tasks (Issues) into Epics.
+* The ability to add Epics into Milestones.
+* A project-level, consolidated Roadmap view that provides visibility across the activities of individuals and Working Groups.
+
+ZenHub was chosen for this proposal for the following reasons:
+
+* It provides the core capabilities of accommodating multiple workstreams, triage/backlog management, and release planning.
+* It provides free accounts for op, en source projects and orgs.

```

Comment:
nitpick (non-blocking): What is "op, en"? I suggest expand the term.

Reply:
It's a typo. :)

I fixed it in the latest commit.